packages/services/storage/src/db/pool.ts

```diff
@@ -16,6 +16,7 @@ export async function getPool(connection: string, maximumPoolSize: number) {
     interceptors: dbInterceptors,
     captureStackTrace: false,
     maximumPoolSize,
+    idleTimeout: 30000,
```

Comment:
Looks like [slonik has `5_000`](https://github.com/gajus/slonik/blob/1228dba2fafcb002a7a78b76864011c08a388453/src/factories/createClientConfiguration.ts#L19) as default value, and [node-postgres has `30_000`](https://github.com/brianc/node-postgres/blob/2a8efbee09a284be12748ed3962bc9b816965e36/packages/pg/lib/defaults.js#L39).

Reply:
Yeah I'm not sure why `slonik` uses another default 🤔